cyences_app_for_splunk/default/macros.conf

```diff
@@ -1088,30 +1082,10 @@ iseval = 0
 definition = search *
 iseval = 0
 
-[cs_palo_ddos_prevented_filter]
```

Comment:
This is for alert, are we removing this alert as well?

Reply:
This is used for the reports.